providers/base/bin/lid_close_suspend_open.sh

```diff
@@ -1,4 +1,19 @@
-#! /usr/bin/bash
+#!/usr/bin/bash
+
+holdoff_timeout_usec=$(gdbus introspect --system --dest org.freedesktop.login1 --object-path /org/freedesktop/login1 -p | grep HoldoffTimeoutUSec | awk '{print $5}' | awk -F\; '{print $1}')
+holdoff_timeout_sec=$(echo "scale=0; $holdoff_timeout_usec / 1000000" | bc)
+
+previous_sleep_log=$(journalctl --output=short-unix --since "$holdoff_timeout_sec seconds ago" -b 0 -r | grep "suspend exit")
+if [[ "$holdoff_timeout_sec" != 0 && "$previous_sleep_log" != "" ]]; then
+    # set the previous_sleep_time
+    previous_sleep_time=$(echo "$previous_sleep_log" | awk -F'.' '{ print $1 }')
+    # sleep a period of time
+    sleep_time=$(( $(date +"%s") - "$previous_sleep_time" ))
+    echo "sleep for $sleep_time seconds ..."
```

Comment:
```suggestion
    echo "DUT was resumed less than 30 seconds ago. Waiting for ${sleep_time seconds} before running the test..."
```

Reply:
clearer expression. But 30 seconds should be ${holdoff_timeout_sec} seconds.